platform/ios/PolyPodApp/PodApi/PostOffice.swift

```diff
@@ -391,13 +391,13 @@ extension PostOffice {
         PodApi.shared.polyNav.setTitle(title: title) { res, error in
         }
     }
-
+    
```

Comment:
Unneeded whitespace?

Reply:
It's due to swift making this automatically -> Wasn't me 🗡️